meson_options.txt

```diff
@@ -38,6 +38,10 @@ option('datarootdir',
         type: 'string',
         value: '',
         description: 'Define the datarootdir for the pkgconf file')
+option('tests',
+        type: 'boolean',
+        value: true,
+        description: 'Build unit tests')
```

Comment:
i'd argue for making this a feature and setting the default value to `enabled`, feels like the more consistent approach with all the other features.